docs/content/blog/2019/12/open-source-introduction.md

```diff
@@ -85,7 +85,7 @@ And even if nobody uses the project code in its original form, developers will b
 ### Learn by teaching
 
 Just moving the code into open source is not enough. 
-Practices, approaches, methods of troubleshooting, and making decisions — this is what is essential. 
+Practices, approaches, methods of troubleshooting, and making decisions — this is what is essential.
```

Comment:
additional space is probably needed to express newline, check pls

Reply:
Only two spaces at the end give a line break.